.changeset/great-roses-pump.md

```diff
@@ -0,0 +1,17 @@
+---
+'@backstage/plugin-catalog': patch
+---
+
+Previously, the color of the Entity Context Menu (in the Entity Page Header) was hardcoded as 'white'. This was an issue for themes that use a header with a white background. The color of the icon can now be overridden in the theme (PluginCatalogEntityContextMenu > button > color). Example:
+
+export function createThemeOverrides(theme: BackstageTheme): Overrides {
```

Comment:
Is this changeset still correct? Also could probably do with some formatting here to make it pretty? And wrapping it in backticks like this:

```
```typescript
```

Reply:
Good point, I have adapted.

Reply:
Good point, I have improved the change set.